bindings/protocols/profinet/index.html

```diff
@@ -0,0 +1,1040 @@
+
+        <tr>
+
+        <tr>
+            <td><code>profv:datalength</code></td>
+            <td>used for aplication and network optimization to appropriately define the specific data length of the resource payload.</td>
+            <td>optional</td>
+            <td><a href="http://www.w3.org/2001/XMLSchema#integer">integer</a></td>
+
+        <tr>
+        
+                </tbody>
+            </table>
+        </section>
+        <section>
+            <h3>Form terms</h3>
+            <table class="def">
+                <thead>
+                    <tr>
+                        <th>Vocabulary term</th>
+                        <th>Description</th>
+                        <th>Assignment</th>
+                        <th>Type</th>
+                    </tr>
+                </thead>
+                <tbody>
+                    <!-- Generate forms terms from the ontology-->
+                    
+        <tr>
+            <td><code>profv:mostSignificantByte</code></td>
+            <td>When <code>true</code>, it describes that the byte order of the data in the PROFINET message is the most significant byte first (i.e., Big-Endian). When <code>false</code>, it describes the least significant byte first (i.e., Little-Endian).If the term is not provided, the TD processor defaults it to <code>true</code></td>
+            <td>optional</td>
+            <td><a href="http://www.w3.org/2001/XMLSchema#boolean">boolean</a></td>
+
+        <tr>
+        
+
+        <tr>
+            <td><code>profv:mostSignificantWord</code></td>
+            <td>When <code>true</code>, it describes that the word order of the data in the Modbus message is the most significant word first (i.e., no word swapping). When <code>false</code>, it describes the least significant word first (i.e. word swapping). If the term is not provided, the TD processor defaults it to <code>true</code></td>
+            <td>optional</td>
+            <td><a href="http://www.w3.org/2001/XMLSchema#boolean">boolean</a></td>
+            
+
+        <tr>
+
+        <tr>
+            <td><code>profv:method</code></td>
+            <td>When <code>true</code>, it describes that the word order of the data in the Modbus message is the most significant word first (i.e., no word swapping). When <code>false</code>, it describes the least significant word first (i.e. word swapping). If the term is not provided, the TD processor defaults it to <code>true</code></td>
+            <td>optional</td>
+            <td><a href="http://www.w3.org/2001/XMLSchema#string">string</a><p>(one of "READ", "WRITE")</p></td>
+            
+
+        <tr>
+        
+
+        <tr>
+            <td><code>profv:pollingTime</code></td>
+            <td>IO supervisor polling rate in milliseconds. It is recommended to keep the polling rate above IO device send clock time. this will ensure that the cyclic communication will not be interferrupted in anyway.</td>
```

Comment:
```suggestion
            <td>IO supervisor polling rate in milliseconds. It is recommended to keep the polling rate above IO device send clock time. this will ensure that the cyclic communication will not be interrupted in anyway.</td>
```

Reply:
Includes @danielpeintner's typo fix
```suggestion
            <td>IO supervisor polling rate in milliseconds. To ensure that the cyclic communication will not be interrupted, it is recommended to keep this polling rate above the IO device's send clock time.</td>
```